README.md

```diff
@@ -1073,178 +811,166 @@ Other Style Guides
-  - [10.1](#modules--use-them) Always use modules (`import`/`export`) over a non-standard module system. You can always transpile to your preferred module system.
+  - [10.1](#modules--use-them) Always use modules (`import`/`export`) over a non-standard module system.
 
     > Why? Modules are the future, let's start using the future now.
 
-    ```javascript
+    ```typescript
     // bad
     const AirbnbStyleGuide = require('./AirbnbStyleGuide');
     module.exports = AirbnbStyleGuide.es6;
 
-    // ok
-    import AirbnbStyleGuide from './AirbnbStyleGuide';
-    export default AirbnbStyleGuide.es6;
-
     // best
     import { es6 } from './AirbnbStyleGuide';
-    export default es6;
+    export es6;
     ```
 
   <a name="modules--no-wildcard"></a><a name="10.2"></a>
-  - [10.2](#modules--no-wildcard) Do not use wildcard imports.
+  - [10.2](#modules--no-wildcard) Do not use wildcard (unless you're forced to) or default imports/exports.
 
     > Why? This makes sure you have a single default export.
 
-    ```javascript
+    ```typescript
     // bad
     import * as AirbnbStyleGuide from './AirbnbStyleGuide';
 
-    // good
-    import AirbnbStyleGuide from './AirbnbStyleGuide';
+    // bad
+    export default AirbnbStyleGuide; // inside one file
+
+    import AirbnbStyleGuide from './AirbnbStyleGuide'; inside another file
+
+    // good 
+    export AirbnbStyleGuide; // inside one file
+
+    import { AirbnbStyleGuide } from './AirbnbStyleGuide'; // inside another file
     ```
 
   <a name="modules--no-export-from-import"></a><a name="10.3"></a>
   - [10.3](#modules--no-export-from-import) And do not export directly from an import.
 
     > Why? Although the one-liner is concise, having one clear way to import and one clear way to export makes things consistent.
 
-    ```javascript
+    ```typescript
     // bad
-    // filename es6.js
-    export { es6 as default } from './AirbnbStyleGuide';
+    // filename es6.ts
+    export { es6 } from './AirbnbStyleGuide';
 
     // good
-    // filename es6.js
+    // filename es6.ts
     import { es6 } from './AirbnbStyleGuide';
-    export default es6;
+    export es6;
     ```
 
   <a name="modules--no-duplicate-imports"></a>
-  - [10.4](#modules--no-duplicate-imports) Only import from a path in one place.
- eslint: [`no-duplicate-imports`](http://eslint.org/docs/rules/no-duplicate-imports)
+  - [10.4](#modules--no-duplicate-imports) Only import from a path in one place, and each import should be on its own line.
+ 
     > Why? Having multiple lines that import from the same path can make code harder to maintain.
 
-    ```javascript
+    ```typescript
     // bad
-    import foo from 'foo';
+    import { named1 } from 'foo';
     // … some other imports … //
-    import { named1, named2 } from 'foo';
+    import { named2 } from 'foo';
 
-    // good
-    import foo, { named1, named2 } from 'foo';
+    // bad
+    import { named1, named2 } from 'foo';
```

Comment:
thinking in an automation mindset, i think we should try to simplify things as much, adding some rule like 'if more than two imports' would complexify stuff...i would put it at a strict do or don't...unless our automation tools allow us otherwise...